From 7e55a3e8ec27314ba009fff7240c3604e8fa917a Mon Sep 17 00:00:00 2001
From: Aswath Govindraju <a-govindraju@ti.com>
Date: Fri, 20 May 2022 18:49:16 +0530
Subject: [tiL5.10-CON PATCH v2] arm64: dts: ti: k3-am62-main: Disable UHS
 speed modes on MMCSD1 instance

Instabilities were observed on a few SD cards when UHS speed modes were
enabled. Therefore, disable UHS modes by adding the tag no-1-8-v tag, in
sdhci1 device tree node.

Signed-off-by: Aswath Govindraju <a-govindraju@ti.com>
---

changes since v1:
- Dropped cover letter
- Moved the tag to main dtsi, so that the changes gets applied
  across all dts files

link to v1:
- https://serenity.dal.design.ti.com/patchwork/project/linux-patch-review/list/?series=2340

 arch/arm64/boot/dts/ti/k3-am62-main.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am62-main.dtsi b/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
index 7bbfcb158842..46bc1adb3fa1 100644
--- a/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
@@ -425,6 +425,7 @@
 		ti,itap-del-sel-sdr25 = <0x0>;
 		ti,clkbuf-sel = <0x7>;
 		bus-width = <4>;
+		no-1-8-v;
 	};
 
 	sdhci2: mmc@fa20000 {
-- 
2.17.1

